AutoMLOps/AutoMLOps.py

```diff
@@ -287,6 +303,7 @@ def _create_default_config(af_registry_location: str,
         schedule_pattern: Cron formatted value used to create a Scheduled retrain job.
         vpc_connector: The name of the vpc connector to use.
     """
+    project_number = str(subprocess.check_output(f'''gcloud projects describe {project_id} --format "value(projectNumber)"''', shell=True, stderr=subprocess.STDOUT)).replace('b\'', '').replace('\\n\'', '')
```

Comment:
We should discuss this - it might be better to make this a parameter the user inputs. Having it run a gcloud command to get this in this could make it more brittle in the generate portion